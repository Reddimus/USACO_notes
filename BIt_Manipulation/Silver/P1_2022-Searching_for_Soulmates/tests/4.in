10
65808 68354
46074 19865
47989 48662
92841 65547
12078 34901
22294 24660
86982 48228
78284 57659
97083 40694
65120 52682
