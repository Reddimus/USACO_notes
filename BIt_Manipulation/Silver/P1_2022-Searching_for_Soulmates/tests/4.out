28
37
25
40
34
30
33
41
43
39
